web/src/components/organisms/StaffSelect.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Select } from 'flowbite-react';
+
+const StaffSelect = (props) => {
```

Comment:
We want this to be more dynamic and call this component as reusable
StaffSelect and StatusSelect declines the purpose of what ReactJs is all about: "To create reusable component, make your development way more faster.
```suggestion
const SelectInput= (props) => {
```